WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -183,6 +188,25 @@ public static void setStatsTimeframe(StatsTimeframe timeframe) {
         }
     }
 
+    public static CommentStatus getCommentsStatusFilter() {
+        int idx = getInt(DeletablePrefKey.COMMENTS_STATUS_TYPE_INDEX);
+        CommentStatus[] commentStatusValues = CommentStatus.values();
+        if (commentStatusValues.length < idx) {
+            return commentStatusValues[0];
+        } else {
+            return commentStatusValues[idx];
+        }
+    }
+    public static void setCommentsStatusFilter(CommentStatus commentstatus) {
```

Comment:
Coming to think of it, it might even be a good idea to remember the Comments filter set for each blog, rather than resetting it each time. A bit overkill though